contingency/contingency-api/src/main/java/com/powsybl/contingency/tasks/AbstractInjectionTripping.java

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright (c) 2016, All partners of the iTesla project (http://www.itesla-project.eu/consortium)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.contingency.tasks;
+
+import com.powsybl.computation.ComputationManager;
+import com.powsybl.iidm.network.Network;
+import com.powsybl.iidm.network.Switch;
+import com.powsybl.iidm.network.Terminal;
+
+import java.util.Objects;
+import java.util.Set;
+
+/**
+ * @author Teofil Calin BANC <teofil-calin.banc at rte-france.com>
+ */
+public abstract class AbstractInjectionTripping extends AbstractTrippingTask {
```

Comment:
Maybe the "id" field could also be shared in this abtract class.

Reply:
done